### PR TITLE
ESLint: cleanup `await expect` from protractor tests

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

the `@typescript-eslint/await-thenable` prevents modified e2e files to be checked in unless all the `expect await` errors are fixed

**What is the new behaviour?**

- cleanup old protractor tests to respect the `@typescript-eslint/await-thenable` eslint rule
- fix errors with double quotes in strings (e2e)
- fix code formatting in various old e2e files
- fix missing documentation tags in e2e utility functions

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/testing/.eslintrc.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
             "accessibility": "explicit"
           }
         ],
-        "@typescript-eslint/no-floating-promises": "error",
+        "@typescript-eslint/no-floating-promises": "off",
```
Comment (DaryaBalvanovich) [inline lib/testing/.eslintrc.json:35]: can we remove this rule?

**Review by DaryaBalvanovich — APPROVED** [review]
Great refactor :slightly_smiling_face:

File: lib/testing/src/lib/protractor/core/utils/file-browser.util.ts [hunk 2]
```diff
@@ -20,14 +20,15 @@ import * as fs from 'fs';
...
-        const file = await browser.driver.wait(() => fs.existsSync(path.join(DEFAULT_ROOT_PATH, 'downloads', fileName)), 30000,`${fileName} not downloaded`);
-
-        await expect(file).toBe(true, `${fileName} not downloaded`);
+        const file: boolean = await browser.driver.wait(
+            () => fs.existsSync(path.join(DEFAULT_ROOT_PATH, 'downloads', fileName)),
+            30000,
+            `${fileName} not downloaded`
+        );
 
+        expect(file).toBe(true);
```
[the remark this answers is not in the document]
Comment (DenysVuika) [reply lib/testing/src/lib/protractor/core/utils/file-browser.util.ts:31]: the file is boolean, so just check for `true`, the second parameter like `expect(true, 'some message')` is deprecated in protractor, and should not be used

File: lib/testing/.eslintrc.json (hunk 1 above)
Comment (DenysVuika) [reply lib/testing/.eslintrc.json]: good point, will try to remove it

**Review by tamaragruszka — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**